src/components/Canvas.vue

```diff
@@ -106,11 +111,15 @@ export default {
     },
     Object.assign.apply(null, [{}, ...Object.values(nodeTypesFlat)])
   ),
+  props: {
+    nodes: { type: Array, default: () => [] },
+    edges: { type: Array, default: () => [] }
+  },
```

Comment:
This allows for the actual instances to be kept outside the canvas (as is done in `groups`).